js/RecentDonations.js

```diff
@@ -0,0 +1,196 @@
+(function() {
+var app = angular.module('RecentDonations', []);
+app.controller('RecentDonationsControl', function($scope) {
+
+    var urlVars = GLOBALS.getUrlVars();
+    var count = Number(urlVars['count']);
```

Comment:
I tried this, but zero is falsy, so setting count = 0 made it = 5. I don't know why you'd *need* zero, but that inconsistency just made me annoyed enough to write a second line